### ISSUE TITLE
Changing color and increasing the username on the note

### ISSUE THREAD

**Issue by IlanAdivi1** [oliviervalentin/moodle-mod_stickynotes#29]
Hello Olivier,

There are a few things that our team has seen that need fixing in the Sticky-notes you've developed:

1. Changing color due to accessibility issues.
2. Increasing the username on the note (font-size) - it's currently very small.

It currently does not meet the accessibility requirements.

https://artzi.lms.education.gov.il/mod/stickynotes/view.php?id=1769290

![username on the note](https://github.com/oliviervalentin/moodle-mod_stickynotes/assets/156344815/c315d818-0fb3-4873-bf8a-0a98e136395c)

Thanks,
Ilan

**Comment by oliviervalentin** [comment]
Hello Ilan,
thank you for this useful comment !
I will search how to adapt font color to note background.
Best regards, 
Olivier

**Comment by oliviervalentin** [comment]
Hello !
I'm working on introducing in CSS a contrast style. This is teh result I obtain : do you think contrast is better ?
I changed colors in my settings : can you give me your hexadecimal color codes so I can test it ?
![image](https://github.com/oliviervalentin/moodle-mod_stickynotes/assets/25584834/31fdac93-8d41-467b-93d5-140e3a818bf6)
Olivier

**Comment by oliviervalentin** [comment]
Hi Ilan,
CSS wasn't the solution...
I have increased text size for username, and added a new function that will calculate contrast. 
Text will be black or white only, but it will be chosen in function of background color. 
Could you give, for an example, your background color so I can test if contrast is alright ?
Thanks !
Olivier

**Comment by IlanAdivi1** [comment]
Hello Olivier,

That's how we think it should be with all of the background colors.

Also, adding style for Username to increase font size, and change color:
1) .headernote class:

font-weight: 650;
font-size: 1rem;
color: #000;

2) .contentnote class:
font-size: 1.2rem;

![WhatsApp Image 2024-02-15 at 15 44 27](https://github.com/oliviervalentin/moodle-mod_stickynotes/assets/156344815/748b9551-f754-43e7-ad0c-8a1ede398d83)

https://artzi.lms.education.gov.il/mod/stickynotes/view.php?id=1769290

Maybe it's a good idea to put the username and note caption color as an option in system setting.

Thanks,
Ilan

**Comment by oliviervalentin** [comment]
I've changed for text size in header and content note.
For now, I will maintain this new function to swtich texte to black or white depending on background color. Let's see if it will fit community or if a new setting is more efficient.
Thank you again for all your help ! :)
Best regards,
Olivier
